android/src/main/java/com/appgoalz/rnjwplayer/RNJWPlayer.java

```diff
@@ -13,6 +14,19 @@ public RNJWPlayer(Context var1, PlayerConfig var2) {
         super(var1, var2);
     }
 
+    @Override
+    public boolean dispatchKeyEvent(KeyEvent event) {
```

Comment:
Well when you install react-navigation and get payer fullscreen then you click on back button, then this method is not called because react-native handle back button event, then is dispatched to react-navigation and then is stops bubbling further. That means that setFullscreenExit method is not called and mess comes up. There are two possible solutions:
1. Remove all events from activity at fullscreen and bind it back on fullscreen exit.
2. onDestroy add fullscreen exit method..

Reply:
Got it, thank you. Updated.